concurrency-limits-grpc/src/test/java/com/netflix/concurrency/limits/grpc/client/ConcurrencyLimitClientInterceptorTest.java

```diff
@@ -72,4 +197,11 @@ public void simulation() throws IOException {
             ClientCalls.futureUnaryCall(channel.newCall(METHOD_DESCRIPTOR, CallOptions.DEFAULT), "request");
         }
     }
+
```

Comment:
We should add helpers to the gRPC builders to make it easy to configure bypass methods.